certbot/examples/cli.ini

```diff
@@ -24,3 +24,15 @@ rsa-key-size = 4096
 # path to the public_html / webroot folder being served by your web server.
 # authenticator = webroot
 # webroot-path = /usr/share/nginx/html
+
+# Uncomment to Automatically agree to the terms of service of the ACME server
+# agree-tos = true
+
+
+# An example of using an alternate ACME server that uses EAB credentials
+# dont just uncomment this - you must also change server URL and keys to match your environment! :)
+
```

Comment:
```suggestion
```

I would hope so :) ! However, since we don't have such a  disclaimer for `email`, let's not do it here either.

Let's also remove the empty line between here and `# server`.